beacon_node/beacon_chain/src/data_availability_checker.rs

```diff
@@ -219,21 +235,14 @@ impl<T: BeaconChainTypes> DataAvailabilityChecker<T> {
     }
 
     /// Put a list of blobs received from the EL pool into the availability cache.
-    ///
-    /// This DOES NOT perform KZG verification because the KZG proofs should have been constructed
-    /// immediately prior to calling this function so they are assumed to be valid.
-    pub fn put_engine_blobs(
+    pub fn put_engine_blobs<O: ObservationStrategy>(
```

Comment:
This could potentially be merged with `put_gossip_blob` given they have the same logic?

Reply:
good catch!